### PR TITLE
Fixtures: Threshold = 2 and Related Attack

### PR DESCRIPTION
**Pull request by davidstrauss**
This is to support testing any fix to #154.

### CODE REVIEW

File: generate_fixtures.py [hunk 1]
```diff
@@ -186,11 +187,51 @@ def __init__(self):
...
+        (public_timestamp_key_2, private_timestamp_key_2) = self.write_and_import_keypair(
+            'timestamp_2')
+        self.repository.timestamp.add_verification_key(public_timestamp_key_2)
+        self.repository.timestamp.load_signing_key(private_timestamp_key_2)
+        self.repository.timestamp.threshold = 2
+        self.repository.mark_dirty(['timestamp'])
+        self.write_and_publish_repository(export_client=True)
+
+
+class TUFTestFixtureThresholdTwoAttack(TUFTestFixtureBase):
```
Comment (pwolanin) [inline generate_fixtures.py:202]: is this supposed to extend TUFTestFixtureThresholdTwo ?

Comment (pwolanin) [reply generate_fixtures.py:202]: I think extended didn't work as desired - instead I copied some of the code below

Comment (davidstrauss) [reply generate_fixtures.py:202]: > is this supposed to extend TUFTestFixtureThresholdTwo ?

Sharp eye. I'll fix this.

Comment (davidstrauss) [reply generate_fixtures.py]: I switched it to extend the other class. The output looks correct to me.

**Review by pwolanin — APPROVED** [review]
Looks good  - tested locally

**Pull request merged**